src/main/java/com/uploadcare/api/Image.java

```diff
@@ -0,0 +1,118 @@
+package com.uploadcare.api;
+
+import com.uploadcare.data.FileData;
+import com.uploadcare.data.ImageInfo;
+
+import java.util.Date;
+
+/**
+ * Created by Egor Petushkov
+ * on 09.06.17.
+ */
+public class Image extends File {
+    public static final int UNKNOW_DPI = -1;
+    public enum Orientation {
```

Comment:
But do you need those as lib user? I don't feel that way at the moment.
If you're going to work with orientation value, you already should know what that means.